src/software/thunderscope/robot_diagnostics/chicker_widget.py

```diff
@@ -69,6 +73,17 @@ def __init__(self, proto_unix_io: ProtoUnixIO) -> None:
             int(DiagnosticsConstants.CHIP_DISTANCE_STEPPER),
         )
 
+        (
+            self.pulse_width_slider_layout,
+            self.pulse_width_slider,
+            self.pulse_width_label,
+        ) = common_widgets.create_slider(
+            "Power (Pulse Width)",
+            DiagnosticsConstants.MIN_PULSE_WIDTH,
+            DiagnosticsConstants.MAX_PULSE_WIDTH,
+            DiagnosticsConstants.PULSE_WIDTH_STEPPER,
+        )
```

Comment:
nit: this isn't necessary but would be nice if the pulse width slider hid itself when we're in m/s Power mode, and vice versa with the other sliders